src/simpleq/queues.py

```diff
@@ -65,12 +66,16 @@ def __init__(self, name, sqs_resource=None):
         if sqs_resource:
             self.sqs_resource = sqs_resource
         else:
-            self.sqs_resource = boto3.resource(
-                "sqs",
-                aws_access_key_id=os.environ["AWS_ACCESS_KEY_ID"],
-                aws_secret_access_key=os.environ["AWS_SECRET_ACCESS_KEY"],
-                region_name=os.environ["AWS_REGION"],
-            )
+            resource_args = {
+                "service_name": "sqs",
+                "aws_access_key_id": os.environ["AWS_ACCESS_KEY_ID"],
```

Comment:
Should we be getting these from settings, rather than directly from `os.environ`?